### PR TITLE
Import of CampaignContacts

### PR DESCRIPTION
**Pull request by nczeranka**
CampaignContact adjustments
- adding fields in CampaignContacts: customerSupplierId, zipcode, pobox
- enabled editing in CampaignContact editor
- passing CustomerSupplierId into UserOnboardData
- renamed label Address to Street/Straße

### CODE REVIEW

File: src/server/db/migrations/20170727130000-contact-customersupplierId.main.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ *
+ * @param {object} data - [Sequelize]{@link https://github.com/sequelize/sequelize} instance.
+ * @param {object} config - A model property for database models and everything from [config.data]{@link https://github.com/OpusCapitaBusinessNetwork/db-init} passed when running the db-initialization.
+ * @returns {Promise} [Promise]{@link http://bluebirdjs.com/docs/api-reference.html}
+ * @see [Applying data migrations]{@link https://github.com/OpusCapitaBusinessNetwork/db-init#applying-data-migrations}
+ */
+module.exports.up = function(db, config)
+{
+    const customerSupplierId = db.queryInterface.addColumn('CampaignContact', 'customerSupplierId', {
+        type: Sequelize.STRING(150),
```
Comment (kwierchris) [inline src/server/db/migrations/20170727130000-contact-customersupplierId.main.js]: The model says it is STRING(30). What is correct?

File: src/server/db/models/Campaign.js [hunk 2]
```diff
@@ -183,6 +183,16 @@ module.exports.init = function(sequelize)
         is: ["[a-zA-Z_\\-0-9]+"]
       }
     },
+    customerSupplierId:
```
Comment (kwierchris) [inline src/server/db/models/Campaign.js]: I guess customerSupplierId, zipCode and pobox should be in **CampaignContact** right?

Comment (nczeranka) [inline src/server/db/models/Campaign.js]: It is part of CampaignContact. CampaignContact starts in line 81.

Comment (kwierchris) [inline src/server/db/models/Campaign.js]: Sorry

**Pull request merged**